src/main/java/de/metas/ui/web/order/sales/purchasePlanning/view/PurchaseRow.java

```diff
@@ -74,39 +74,46 @@
 			@ViewColumnLayout(when = JSONViewDataType.includedView, seqNo = 20)
 	})
 	private final JSONLookupValue vendorBPartner;
-
-	@ViewColumn(captionKey = "QtyToDeliver", widgetType = DocumentFieldWidgetType.Quantity, layouts = {
+	
+	
+	@ViewColumn(captionKey = "QtyAvailable", widgetType = DocumentFieldWidgetType.Quantity, layouts = {
```

Comment:
QtyAvailalable is a bit unspecific..since we already use the term and ADElement  `Qty_AvailableToPromise`
Material Disposition window (table the table is currently called `MD_Candidate`), please go with `Qty_AvailableToPromise`.